beacon-chain/sync/service.go

```diff
@@ -80,13 +81,18 @@ func (ss *Service) Stop() error {
 // Status always returns nil.
 // TODO(1206): Add service health checks.
 func (ss *Service) Status() error {
+	if ss.failStatus != nil {
```

Comment:
Is this the only check you are going to add?

What about when the client is not synced? I think that is an unhealthy condition

Reply:
Added TestStatus() and unhealthy status for unsynced service!